### PR TITLE
refactor(package): more generic package name

### LINKED ISSUE #14: Should the package name be `eslint-find-rules`
Should the package name be `eslint-find-rules`? Similar to its predecessor `eslint-find-new-rules`?

Not sure, but I guess 
`eslint-find-rules [option] <filename>`
it sounds better than
`eslint-rule-finder [option] <filename>`

a verb seems better than a noun, in this case.

Thoughts?

### PR DESCRIPTION
**Pull request by sarbbottam**
fixed #14

### CODE REVIEW

**Comment by sarbbottam** [comment]
@ta2edchimp please take a look. Thought it's minor change, extra pair of 👀 always helpful.

**Comment by ta2edchimp** [comment]
LGTM.
The Repo (and its ReadMe) have to be updated too ... again.

---

>Thought it's minor change, extra pair of :eyes: always helpful.

Definitely. I think it's always good to let someone other review ones own code!

**Comment by sarbbottam** [comment]
>The Repo (and its ReadMe) have to be updated too ... again.

Let me update the `readme` with this PR, and update the description once it is merged.

>>Thought it's minor change, extra pair of :eyes: always helpful.
>Definitely. I think it's always good to let someone other review ones own code!

Was aware of the repo desc, but missed the `readme` part, this is why review is so much valuable.

File: README.md [hunk 1]
```diff
@@ -1,15 +1,13 @@
-# eslint-rule-finder
-
-[![Join the chat at https://gitter.im/sarbbottam/eslint-rule-finder](https://badges.gitter.im/sarbbottam/eslint-rule-finder.svg)](https://gitter.im/sarbbottam/eslint-rule-finder?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (sarbbottam) [inline README.md]: Will create a new `gitter` room with new repo name.

**Comment by sarbbottam** [comment]
@ta2edchimp please have a look when you get time.

**Comment by ta2edchimp** [comment]
LGTM!

**Pull request merged**